operator_ui/src/components/Link.tsx

```diff
@@ -27,7 +27,7 @@ const styles = (_theme: Theme) =>
   })
 
 interface IProps extends WithStyles<typeof styles> {
-  children: React.ReactNode
+  children: React.ReactNode | string
```

Comment:
isn't `| string` redundant since `React.ReactNode` covers string (i could be wrong)?